packages/blocks/src/api/validation/index.js

```diff
@@ -710,14 +719,16 @@ export function isEquivalentHTML( actual, expected, logger = createLogger() ) {
  * @return {Object} Whether block is valid and contains validation messages.
  */
 
+// TODO ^ What is this?
+
```

Comment:
This is a duplicate JSDoc comment with a different signature. Is this an overload? Is that possible with JSDocs?

Reply:
TypeScript only very recently [added support](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#overload-support-in-jsdoc) for overloads via `@overload`, so it's not that.

It's a typo introduced during a refactor:

https://github.com/WordPress/gutenberg/commit/984160d1368695f22222aea4e20527d7699b842c#diff-ee4697a721079b9d6da3660250b599d45ebdf68c95a055f15f618d38ba73014a

Reply:
Thanks for the confirmation. I actually just learned about that and did a few `@overloads` on a PR to `hpq` today 😄